lib/spack/docs/packages_yaml.rst

```diff
@@ -0,0 +1,549 @@
+.. Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+   Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+   SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+.. _build-settings:
```

Comment:
The name of this link seems inconsistent now that the file is not in `build_settings.rst`